includes/functions/content-sync.php

```diff
@@ -311,3 +315,58 @@ function get_post_data( $post ) {
 	 */
 	return apply_filters( 'sophi_post_data', $data );
 }
+
+/**
+ * Check if the current data has been already tracked recently ans should be skipped.
+ *
+ * @since 1.3.0
+ *
+ * @param array $data Data being tracked
+ *
+ * @return boolean
+ */
+function maybe_skip_track_event( $data ) {
```

Comment:
You may be able to simplify this to test if the request is coming from the legacy meta box loader:

```php
$skip = false;
if ( isset( $_GET['meta-box-loader'] ) && $_GET['meta-box-loader'] ) ) {
  $skip = true;
}
```

I haven't tested this so you might want to test a couple of things:
* it works as expected
* the code doesn't affect the classic editor